media-libs/mesa/mesa-9999.ebuild

```diff
@@ -99,10 +100,34 @@ RDEPEND="
 	unwind? ( sys-libs/libunwind[${MULTILIB_USEDEP}] )
 	llvm? (
 		$(llvm_gen_dep "
-			llvm-core/llvm:\${LLVM_SLOT}[llvm_targets_AMDGPU(+),${MULTILIB_USEDEP}]
```

Comment:
The AMDGPU use dependency was intentionally made unconditional in commit 6c4bdcbd3b25bdc3b4ccd6ab7d839840dbbef813.

```
commit 6c4bdcbd3b25bdc3b4ccd6ab7d839840dbbef813
Author: Matt Turner <mattst88@gentoo.org>
Date:   Thu May 11 16:04:14 2023 -0400

    media-libs/mesa: Simplify LLVM dependencies
    
    Specifically, just always require that sys-devel/llvm (or
    sys-devel/clang) is built with LLVM_TARGETS=AMDGPU. It's been in
    package.use.force since llvm:13 and seems unlikely to be removed. It
    also massively simplifies the dependencies.
    
    Signed-off-by: Matt Turner <mattst88@gentoo.org>
```

If we want to revisit this decision, we can, but we shouldn't do it in this PR (unless it's required for some reason that I don't understand)